hpOneView/resources/resource.py

```diff
@@ -186,6 +191,20 @@ def create(self, resource, timeout=-1):
 
         return self._task_monitor.wait_for_task(task, timeout)
 
+    def create_with_uri(self, resource, uri, timeout=-1):
+        if not resource:
```

Comment:
Why don't you just add an optional uri argument to our current create function? Like update function does.
We could have an unique implementation.
It may be some tests would break :(